### ISSUE TITLE
Fix list alignment issue

### ISSUE THREAD

**Issue by brandonhaslegs** [drips-network/app#714]
It shouldn't overflow like this
<img width="768" alt="image" src="https://github.com/drips-network/app/assets/4406983/4bb05490-a300-4b39-8f0b-2cd36ed4136b">

**Comment by brandonhaslegs** [comment]
@evvvritt there's plenty of space here so something seems wrong.

**Comment by evvvritt** [comment]
right, but do you think we should truncate (...) the address or should be able to scroll to read it completely?

**Comment by brandonhaslegs** [comment]
I think truncating is better. With click/tap to copy address.
